compiler/src/iree/compiler/Dialect/Util/Transforms/test/test_integer_divisibility_analysis.mlir

```diff
@@ -0,0 +1,157 @@
+// RUN: iree-opt --split-input-file --iree-util-test-integer-divisibility-analysis --allow-unregistered-dialect %s | FileCheck %s
```

Comment:
I searched for divisibility related test out of curiosity and find a `integer_divisibility.mlir`. That pass is already invoking `--iree-util-optimize-int-arithmetic`. Why do you need to invent this new test flag instead of relying on that one, but to add more affine.apply test cases there?